README.md

```diff
@@ -8,9 +8,9 @@ Ballerina Persist - SQL Library
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-persist.sql.svg)](https://github.com/ballerina-platform/module-ballerinax-persist.sql/commits/main)
 [![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/persist.sql.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fpersist.sql)
 
-This library provides Ballerina `persist` tooling for SQL, which provides functionality to store and query data conveniently through a data model instead of using relational databases.
+This library provides relation database support for the `bal persist` feature, which provides functionality to store and query data conveniently through a data model instead of using relational databases.
```

Comment:
Is this line specific to SQL or in general? if for SQL can we use `SQL query language` instead of `using relational databases`?